pkg/compressor/compressor_tar.go

```diff
@@ -35,9 +36,14 @@ import (
 type Tar struct{}
 
 // Compress compresses the directory into the writer
-func (t *Tar) Compress(directory files.Directory, writer *bytes.Buffer) (e error) {
-	gzipWriter := gzip.NewWriter(writer)
+func (t *Tar) Compress(directory files.Directory, writer *bytes.Buffer) error {
```

Comment:
Yeah I can see how that messed with the local err declaration. I'm fine with it, I just usually like to have specific names such as err added to the return type to clarify the meaning of the return type. Seems fine tho